data/actions/scripts/other/doors.lua

```diff
@@ -1,3 +1,51 @@
+local positions = {
+	{x = 1, y = 0}, -- east
+	{x = -1, y = 1}, -- south
+	{x = -1, y = -1}, -- west
+	{x = 1, y = -1} -- north
+}
+
+local function moveToAptPos(creature, round)
+	local pos = creature:getPosition()
+	for k,v in pairs(positions) do
+		pos.x = pos.x + v.x
```

Comment:
If you use the Position constructor (see above), you can construct the offset position like this:
`local offsetPosition = pos + v`